src/components/tag-selector.yml

```diff
@@ -1,7 +1,4 @@
-linkTextSelectMore:
-  en: + More
-  de: + Mehr
-linkTextSelectFirst:
+linkText:
   en: + Add tag filter
   de: + Tag-Filter hinzufügen
```

Comment:
As this label is directly next to the "x"-Button/Link, we should also adjust the translation `deselectTags` in `search.yml` to "Remove all tag filters" instead of "Remove all tags". What do you think? Otherwise we call these things by two different names.

Reply:
True, it's inconsistent this way.
Dropped the "filters" from ""Remove all tags" as discussed.